src/components/ProcessMoneyRequestHoldMenu.js

```diff
@@ -0,0 +1,106 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@styles/useThemeStyles';
+import variables from '@styles/variables';
+import Button from './Button';
+import Icon from './Icon';
+import * as Illustrations from './Icon/Illustrations';
+import Popover from './Popover';
+import refPropTypes from './refPropTypes';
+import Text from './Text';
+
+const propTypes = {
+    isVisible: PropTypes.bool.isRequired,
+    onClose: PropTypes.func.isRequired,
+    onConfirm: PropTypes.func.isRequired,
+    anchorPosition: PropTypes.shape({
+        horizontal: PropTypes.number,
+        vertical: PropTypes.number,
+    }),
+    anchorRef: refPropTypes,
+};
+
+const defaultProps = {
+    anchorPosition: {},
+    anchorRef: () => {},
+};
+
+function ProcessMoneyRequestHoldMenu({isVisible, onClose, onConfirm, anchorPosition, anchorRef}) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const holdMenuSections = useMemo(() => {
```

Comment:
I think it would be cleaner to write a separate component for `holdMenuSections` instead of keeping it in the `useMemo` hook, especially since it does not depend on any of the props :)

Reply:
Will do!